### PR TITLE
DPC-1075: Admin site separation: add user dashboard

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1075](https://jira.cms.gov/browse/DPC-1075)

- Create a functional user dashboard on the new admin site

### Proposed Changes

Create a functional user dashboard.

Will need to fix the organization assignment when adding the organization dashboard.

### Change Details

Admin users can 
- View all users
- View users based on vendor/provider assigned/unassigned
- View individual users
- Edit individual users (Name, email)
- Filter users
- Download CSV of user list (filtered or not)

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

- Nothing new has been added, we are just moving the `internal user dashboard` from the web app to this new application.

### Acceptance Validation

Screen Shot 2021-02-23 at 2 38 02 PM

Screen Shot 2021-02-23 at 2 38 12 PM

Screen Shot 2021-02-23 at 2 38 21 PM

Screen Shot 2021-02-23 at 3 25 38 PM

### CODE REVIEW

File: dpc-admin/app/assets/stylesheets/base/0_variables.scss [hunk 1]
```diff
@@ -1,8 +1,12 @@
 // Breakpoints
+$layout-breakpoint: 965px;
 $width-nav: 1085px;
 
 // Colors
 $color-gray-ultra-light: #f6f6f6;
+$color-success-alt-lightest: rgba($color-success-lightest, .9);
+$color-warn-alt: #e27100;
```
Comment (SalGutierrez) [inline dpc-admin/app/assets/stylesheets/base/0_variables.scss:8]: good choice, e27100 is a great color.

File: dpc-admin/app/helpers/user_helper.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module UserHelper
+  def city_state(user)
+    user.city + ', ' + user.state
```
Comment (SalGutierrez) [inline dpc-admin/app/helpers/user_helper.rb:5]: can city be empty?

Comment (Sun-Mountain) [reply dpc-admin/app/helpers/user_helper.rb:5]: Not when the user is creating their account in the web app.

But that's not what this helper method is doing. It's putting the city/state in a format that's easier to move around in the front end.

**Review by SalGutierrez — DISMISSED** [review]
Syntactically it LGTM!

**Review by dhgreene — APPROVED** [review]
👍

File: dpc-admin/app/helpers/user_helper.rb (hunk 2 above)
Comment (PatKelsh) [inline dpc-admin/app/helpers/user_helper.rb:5]: Use string interpolation here
`"#{user.city}, #{user.state}"`

**Review by PatKelsh — CHANGES_REQUESTED** [review]
Just one small change

**Review by PatKelsh — APPROVED** [review]
Looks good to me

**Pull request merged**